### ISSUE TITLE
Warn if Sent folder not found

### ISSUE THREAD

**Issue by qoheniac** [thundernest/k-9#6276]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.202

### Problem you are trying to solve

With the default settings if Sent folder detection fails, sent emails aren't stored anywhere.

### Suggested solution

If “Upload sent messages” is checked but ”Sent folder“ is “Automatic (None)” show a warning message when the user tries to send an email.

**Comment by Standard8** [comment]
Please can this be marked as a data loss bug? I think that not saving data without when it is expected, is not an enhancement request

**Comment by theAkito** [comment]
@Standard8 From what you said in https://github.com/thundernest/k-9/issues/6847#issuecomment-1519566097, you are correct.

This is not just an enhancement of an existing feature. Something clearly goes *wrong*, because a very basic functionality like reading your sent mails is missing.

I actually encountered this the hard way. I received access to an organisation e-mail address, sent a couple of more or less important mails & they just "don't exist", because K-9 failed to save them or at least warn me about the missing folder.

**This is a bug.**

**Comment by Standard8** [comment]
I just realised, this also affects the trash folder. So if you accidentally delete something from that nice primary UI button (which I have done in the past), then it is gone... No warning..., disappeared into /dev/null.

Spam and Archives do not have the options show up for those functions which is reasonable.

Save to drafts is disabled... which is reasonable, though again, confusing that there's nothing pointing you as to why you can't save as draft.

**Comment by kapsiR** [comment]
Can confirm, this is a bad thing. 
I suggested to use K-9 to a colleague of mine, which trapped into this issue and lost some emails because of this.

The default setting for `Sent` is "Automatic", which can have a value of `None`.  
If you send an email, there is no warning or anything like that to tell you that the email hasn't been stored anywhere!

**Comment by paginabianca** [comment]
I just encountered this issue as well.
Apparently I got asked to configure the folders when setting up the accounts. I can't remember if that was the case or not as I used the import from Thunderbird desktop functionality.

Some kind of warning would still be nice if the "Upload Sent Messages to Sent folder" option is ticked and "Automatic (None)" is the sent folder.

**Comment by kewisch** [comment]
Agreed. When I first used TfA I thought "oh its automatic, I guess it will just set itself up later". I just recently learned that in Gmail you need to expose the Sent/Drafts/Trash etc folders for IMAP using the checkboxes in the folder settings. If you don't do this, Gmail will not advertise a Sent folder, and Thunderbird won't be able to find it.

I have this written down as a use case for our in-app error notifications, and in addition to an error if things are incorrectly configured, we should have some sort of warning at the folder selector screen so that people are strongly advised to select a folder if it is None.

**Comment by ewerybody** [comment]
How is this issue 3 YEARS OLD?? 😵
I just found the settings for this and bent it all to the **actually existing folders!**

I was haunted several times thinking I didn't send the mails asking the recipient if I did send a mail. ooof.

**Comment by ewerybody** [comment]
Never ever should it happen that a sent mail is simply thrown away!
It should create a folder if there isn't really anything!

yeah and warnings would be nice! 👍

**Comment by hacklschorsch** [comment]
It seems an update to K-9 over two months ago made my client forget the already set folders?

Today I saw with horror that I'm missing 2,5 months of outgoing eMails that I had sent from my Mobile.

Saving sent eMail to /Sent worked before and silently stopped working.

This is a data loss bug.
